tests/test_logging.py

```diff
@@ -134,38 +134,50 @@ def test_configure_extended_logging(self):
         }
         opsdroid.configure_logging(config)
         rootlogger = logging.getLogger()
-        self.assertEqual(len(rootlogger.handlers), 1)
-        self.assertEqual(logging.StreamHandler, type(rootlogger.handlers[0]))
+        assert len(rootlogger.handlers) == 1
+        assert isinstance(rootlogger.handlers[0], logging.StreamHandler)
 
-    def test_configure_default_logging(self):
+    def test_configure_default_logging(self, capsys):
         config = {}
         opsdroid.configure_logging(config)
         rootlogger = logging.getLogger()
-        self.assertEqual(len(rootlogger.handlers), 2)
-        self.assertEqual(logging.StreamHandler, type(rootlogger.handlers[0]))
-        self.assertEqual(rootlogger.handlers[0].level, logging.INFO)
-        self.assertEqual(
-            logging.handlers.RotatingFileHandler, type(rootlogger.handlers[1])
-        )
-        self.assertEqual(rootlogger.handlers[1].level, logging.INFO)
-        self.assertLogs("_LOGGER", "info")
+        assert len(rootlogger.handlers), 2
+        assert isinstance(rootlogger.handlers[0], logging.StreamHandler)
+        assert rootlogger.handlers[0].level == logging.INFO
+        assert isinstance(rootlogger.handlers[1], logging.handlers.RotatingFileHandler)
+        assert rootlogger.handlers[1].level == logging.INFO
+
+        captured = capsys.readouterr()
+        assert "INFO opsdroid.logging: Started opsdroid" in captured.err
```

Comment:
No, it was not working for me :(

Reply:
That's alright I think that's the issue yeah, since we are setting up logging then caplog doesn't seem to be able to get that. 

I might check the lib to see if there is any issue about that, but yeah I'm happy with this work 😄👍